src/components/sbb-header-action/sbb-header-action.e2e.ts

```diff
@@ -1,62 +1,70 @@
-import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
 import { waitForCondition } from '../../global/testing';
+import { assert, expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import { sendKeys } from '@web/test-runner-commands';
+import { EventSpy } from '../../global/testing/event-spy';
+import { SbbHeaderAction } from './sbb-header-action';
 
 describe('sbb-header-action', () => {
-  let element: E2EElement, page: E2EPage;
+  let element: SbbHeaderAction;
 
   beforeEach(async () => {
-    page = await newE2EPage();
-    await page.setContent('<sbb-header-action id="focus-id">Action</sbb-header-action>');
-    await page.waitForChanges();
+    element = await fixture(html`<sbb-header-action id="focus-id">Action</sbb-header-action>`);
+  });
 
-    element = await page.find('sbb-header-action');
+  it('renders', async () => {
+    assert.instanceOf(element, SbbHeaderAction);
   });
 
   describe('events', () => {
     it('dispatches event on click', async () => {
-      await page.waitForChanges();
-      const changeSpy = await page.spyOnEvent('click');
+      await element.updateComplete;
```

Comment:
In this case there is no changes to be awaited (the waiting for initialization is taken care by the `fixture`)
```suggestion
```